src/components/lyne-cta-button/lyne-cta-button.tsx

```diff
@@ -16,21 +19,36 @@ import { InterfaceButtonAttributes } from './lyne-cta-button.d';
 
 export class LyneCtaButton {
 
-  /**
-   * @sampleDocTag sampleName - sampleValue
-   * @sampleDocTag2 sampleName2 - sampleValue2
-   */
-  @Prop() public samplePropForJSDocs = 'sample prop';
-
   /** Label text to show on the button */
-  @Prop() public label = 'Default button text';
+  @Prop() public label? = 'Default button text';
 
   /** Variant of the button, like primary, secondary etc. */
   @Prop() public variant?: InterfaceButtonAttributes['variant'] = 'primary';
 
+  /** Size variant, either large or small. */
+  @Prop() public size?: InterfaceButtonAttributes['size'] = 'large';
+
+  /** Set to true to get a disabled button */
+  @Prop() public disabled? = false;
+
   /** Id which is send in the click event payload */
   @Prop() public eventId?: string;
 
+  /** Define if icon should be shown or not */
+  @Prop() public icon? = false;
+
+  /** Define if icon should be shown or not */
+  @Prop() public iconDescription?: string;
+
+  /** The type attribute to use for the button */
+  @Prop() public type?: InterfaceButtonAttributes['type'] = 'button';
+
+  /** The name attribute to use for the button */
+  @Prop() public name?: string;
+
+  /** The value attribute to use for the button */
+  @Prop() public value?: string;
+
   @Element() private _element: HTMLElement;
```

Comment:
Do we want to get the consumers control over the customEvent config or to certain parts like bubbles?

Reply:
Well, this is the data send along with the form. I think it makes sense, that consumers can define name and value that are send with the form data, right?

Not sure if you highlighted the correct part of the code. Your question seems to be about the `CustomEvent` we are dispatching, right? If so, then i think it would be a bad idea to give consumers control over bubbling and composing of the dispatched event, since i think we must in control over about when to bubble and when not to. Or what do you think? I'd leave it as is and change it as soon as we realize that we have issues with event handling, ok?